app/assets/stylesheets/application.scss

```diff
@@ -26,12 +26,17 @@
   visibility: hidden;
   opacity: 0;
  }
- 
- .js {
+
+.js {
   visibility: visible;
   opacity: 1;
- }
+}
+
+.form-button {
```

Comment:
@sdellis What is this for?

Reply:
These are just indentation fixes, followed by a rule to show the page after all other css rules have loaded.